RPIPs/RPIP-60.md

```diff
@@ -1,52 +0,0 @@
----
-rpip: 60
-title: Protocol Upgrade Guardrails
-description: Introduces a delay after protocol upgrades have been confirmed but prior to them coming into effect.
-author: Valdorff (@Valdorff)
-discussions-to: TBD
-status: Draft
-type: Protocol
-category: Core
-created: 2024-03-08
-requires: 
-tags: tokenomics-2024, tokenomics-content
----
-
-## Abstract
-
-Regularly scheduled protocol upgrades now have a delay between when they are passed, and when they are executed. 
-
-The security council may trigger a faster upgrade with a shorter delay, but there are limitations on when they may use this power. This security council power has a maximum number of uses that may be set by the pDAO. The security council cannot use the fast upgrade power if they have no remaining uses, and each use removes one seal. Seals may be restored via pDAO vote. 
-
-Additionally the security council may veto contract upgrades. The veto power should not be used lightly, and misuse may result in replacement of the security council by the pDAO.
-
-## Specification
-
-### Protocol upgrades
-- All protocol upgrades SHALL have a `default_upgrade_delay` delay between when they are passed and when they are executed
-- The security council SHALL have a limited-use power to use a shorter delay of `fast_upgrade_delay`
-  - This power SHALL be usable if `fast_upgrade_seal_count` > 0 
-  - `fast_upgrade_seal_count` SHALL be decremented by one upon using this power
-  - The pDAO MAY include a change to `fast_upgrade_seal_count` in a voted upgrade*
-  - The pDAO MAY change `fast_upgrade_seal_count` via vote**
-  - The security council SHALL use this power if and only if:
-    - A vote specifically requested a fast upgrade
-    - The vote passed with a supermajority of `fast_upgrade_supermajority` or more
-    - The security council deems it acceptable from a security point of view
-- The security council SHALL have the power to veto contract upgrades (which are proposed by the oDAO)
-  - Veto powers SHALL NOT be used lightly and SHALL be reserved for cases of vote manipulation, malicious action (eg, oDAO votes in a proposal against the pDAO governance), or contract upgrades that would result in clear damage to the Rocket Pool project
-  - When this veto power is exercised, the security council MUST publish a Veto Explanation Document that describes why this step was taken; if possible, the report SHOULD also suggest potential similar-but-non-damaging votes that could be considered
-  - If this veto power is abused, the pDAO SHOULD consider replacing the security council
-- The initial settings SHALL be:
-  - `default_upgrade_delay`: 3 weeks
-  - `fast_upgrade_delay`: 3 days
```

Comment:
Probably too slow. I would rather see like 24 hours alongside a unanimous requirement.

Reply:
Right - this can't handle real emergencies. Some of the "pause" style powers may be able to. This would be more like "nobody can claim rewards -- it's super annoying, but the rewards are still all there to be claimed once resolved" or perhaps "fix the core issue so we can unpause"